.gitignore

```diff
@@ -127,3 +127,22 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
```

Comment:
To be clear: Is everything in this commit taken (without changes) from `oresat-live-software/camera`?

Reply:
I specified then directories oresat_dxwifi/camera but yeah these are all from the gitignore of oresat-live-software/camera